KeyVaultAuthSample.cs

```diff
@@ -60,26 +55,25 @@ private async Task authUsingADALCallbackAsync(string vaultBaseURL)
             Console.WriteLine("Authenticating to Key Vault using ADAL callback.");
             Console.WriteLine(vaultBaseURL);
 
-            // Set up a KV Client with an ADAL authentication callback function
-            KeyVaultClient kvClient = new KeyVaultClient(
-                async (string authority, string resource, string scope) =>
+            var credential = new DefaultAzureCredential();
+            SecretClient secretClient = new SecretClient(new Uri(vaultBaseURL), credential);
+
+            // Set and get an example secret
+            await secretClient.SetSecretAsync("test-secret", "test-secret-value-using-adal");
+
+            await foreach (SecretProperties secret in secretClient.GetPropertiesOfSecretsAsync())
```

Comment:
Is there some reason you're getting all secrets? This is against best practices of getting only what you need, if you know up front. This is a practice we've been trying to get customers away from with newer samples.

Reply:
@heaths - I have updated, please review.